services/webdav/pkg/config/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 
 	HTTP HTTP `yaml:"http"`
 
+	DisablePreviews      bool            `yaml:"disablePreviews" env:"OCIS_DISABLE_PREVIEWS;WEBDAV_OPTION_DISABLE_PREVIEWS" desc:"Set this option to 'true' to disable all preview related webdav paths."`
```

Comment:
```suggestion
	DisablePreviews      bool            `yaml:"disablePreviews" env:"OCIS_DISABLE_PREVIEWS;WEBDAV_DISABLE_PREVIEWS" desc:"Set this option to 'true' to disable all preview related webdav paths."`
```

Reply:
I am aware of this, this was on purpose to be en par with: `WEB_OPTION_DISABLE_PREVIEWS`

Reply:
Yeah I was thinking that. :rofl: But imo `WEB_OPTION` is the prefix and `DISABLE_PREVIEWS` is the option

/cc @mmattel to the rescue pls :D

Reply:
`WEBDAV_DISABLE_PREVIEWS` is fine IMO